pkg/driver/utils.go

```diff
@@ -106,3 +107,29 @@ func getCapacity(cvc *apisv1.CStorVolumeConfig) string {
 	qCap := cvc.Spec.Capacity[corev1.ResourceStorage]
 	return qCap.String()
 }
+
+func getVolumeCondition(vol *apisv1.CStorVolume) *csi.VolumeCondition {
+	condition := &csi.VolumeCondition{}
+	if vol.Status.Phase != apisv1.CVStatusHealthy {
+		condition.Abnormal = true
+	}
+
+	switch vol.Status.Phase {
+	case apisv1.CVStatusHealthy:
+		condition.Message = "Volume status is up"
```

Comment:
How about below? If yes similarly in other places also
```suggestion
		condition.Message = "Volume is healthy"
```